src/haxelib/server/Repo.hx

```diff
@@ -80,6 +80,15 @@ class Repo implements SiteApi {
 		};
 	}
 
+	public function getLatestVersion( project : String ) : SemVer {
+		var p = Project.manager.select($name == project);
+		if( p == null )
+			throw "No such Project : "+project;
+
+		var vl = Version.manager.search($project == p.id, { orderBy: [-major, -minor, -patch, -previewNum], limit: 1});
```

Comment:
Gotta check SPOD macros, maybe enum indices are used, then you can sort on them easily.